src/app/clusters/barrier-control-server/barrier-control-server-cli.c

```diff
@@ -37,6 +37,7 @@
  *******************************************************************************
  ******************************************************************************/
 
+#if 0
```

Comment:
This is because we don't actually want to build this file, right?  At the very least we should document that.  Better yet would be to actually not build this file.  I thought @rwalker-apple had some plan for that; worth checking.